frontend/pages/_app.tsx

```diff
@@ -92,7 +95,7 @@ const App: NextPage<AppProps> = ({ Component, pageProps, err }: ExtendedAppProps
           </PageTransitionContextProvider>
         </SessionKiller>
       </Provider>
-    </>
+    </CacheProvider>
```

Comment:
Do we still need `StyledEngineProvider` and `StylesProvider` above?

Reply:
After reading the documentation for both, it looks like we should keep `StyledEngineProvider` while we're using JSS (which we should move away from going forward), but I don't think we need `StylesProvider`.

Just made some changes and tested on a local production build; SSR seems to work fine with JS disabled.